### DISCUSSION TITLE
Unable to reverify academic status via email.

### DISCUSSION THREAD

**Discussion by kyee-rs** [community/community#46349] [GitHub Education]
Hello, Jan 31st i lost my GitHub Education pack due to reverification needs. Since i have no student ID but only email, i can only verify it this way, but the system asks only for student ID giving no other choice.

**Comment by jge162** [comment]
You must should reach out to [GitHub Education support](https://support.github.com/request/education?tags=hubberfy_education) to help you resolve your recertification. We can only discuss it here. Good luck.
